hashicorp/waypoint
# ui: update favicon to match website

## Why the change?

I was finding it tricky to identify my Waypoint server tabs in amongst other HashiCorp pages.

## What does it look like?

| Before | After |
| --- | --- |
| <img width="251" alt="CleanShot 2021-10-13 at 15 56 19@2x" src="https://user-images.githubusercontent.com/34030/137147385-69a4269a-2d17-4b15-8ef8-36ceb9885213.png"> | <img width="251" alt="CleanShot 2021-10-13 at 15 56 44@2x" src="https://user-images.githubusercontent.com/34030/137147405-9e718b7b-6646-49f3-bf8e-a1d4b3808001.png"> |

## How do I test it?

1. Check out the branch `git checkout ui/favicon-update`
2. Boot the dev server `cd ui && yarn start`
3. [Visit localhost:4200](http://localhost:4200)
4. Verify the favicon looks nice (you may need to bust your browser cache)
